apps/web/app/future/reschedule/[uid]/embed/page.tsx

```diff
@@ -1,22 +1,11 @@
 import { getServerSideProps } from "@pages/reschedule/[uid]";
+import OldPage from "@pages/reschedule/[uid]";
 import { withAppDirSsr } from "app/WithAppDirSsr";
-import type { SearchParams } from "app/_types";
-import type { Params } from "next/dist/shared/lib/router/utils/route-matcher";
-import { cookies, headers } from "next/headers";
+import withEmbedSsrAppDir from "app/WithEmbedSSR";
+import { WithLayout } from "app/layoutHOC";
 
-import { buildLegacyCtx } from "@lib/buildLegacyCtx";
-import withEmbedSsr from "@lib/withEmbedSsr";
+const getData = withAppDirSsr(getServerSideProps);
 
-type PageProps = Readonly<{
-  params: Params;
-  searchParams: SearchParams;
-}>;
+const getEmbedData = withEmbedSsrAppDir(getData);
 
-const Page = async ({ params, searchParams }: PageProps) => {
-  const legacyCtx = buildLegacyCtx(headers(), cookies(), params, searchParams);
-  await withAppDirSsr(withEmbedSsr(getServerSideProps))(legacyCtx);
-
-  return null;
-};
-
-export default Page;
+export default WithLayout({ getLayout: null, getData: getEmbedData, Page: OldPage });
```

Comment:
We dont need to use WithLayout here, because we dont need to wrap the page with PageWrapper